datasets/irc_disentangle/README.md

```diff
@@ -0,0 +1,195 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+- structure-prediction
+task_ids:
+- coreference-resolution
+- dialogue-modeling
+---
+
+
+---
+
+# Dataset Card for IRC Disentanglement
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://jkk.name/irc-disentanglement/
+- **Repository:** https://github.com/jkkummerfeld/irc-disentanglement/tree/master/data
+- **Paper:** https://aclweb.org/anthology/papers/P/P19/P19-1374/
+- **Leaderboard:** NA
+- **Point of Contact:** jkummerf@umich.edu
+
+### Dataset Summary
+
+Disentangling conversations mixed together in a single stream of messages is a difficult task, made harder by the lack of large manually annotated datasets. This new dataset of 77,563 messages manually annotated with reply-structure graphs that both disentangle conversations and define internal conversation structure. The dataset is 16 times larger than all previously released datasets combined, the first to include adjudication of annotation disagreements, and the first to include context.
+
+### Supported Tasks and Leaderboards
+
+Conversational Disentanglement, Coreference Resolution, Dialogue Modeling
+
+### Languages
+
+English (en)
+
+## Dataset Structure
+
+### Data Instances
+
+For Ubuntu:
+
+data["train"][1050]
+
+{
+
+  'ascii': "[03:57] <Xophe> (also, I'm guessing that this isn't a good place to report minor but annoying bugs... what is?)", 
+
+  'connections': [1048, 1054, 1055, 1072, 1073], 
+
+  'date': '2004-12-25', 
+
+  'raw': "[03:57] <Xophe> (also, I'm guessing that this isn't a good place to report minor but annoying bugs... what is?)", 
+
+  'tokenized': "<s> ( also , i 'm guessing that this is n't a good place to report minor but annoying bugs ... what is ?) </s>"
+
+}
```

Comment:
````suggestion
```
{

  'ascii': "[03:57] <Xophe> (also, I'm guessing that this isn't a good place to report minor but annoying bugs... what is?)", 

  'connections': [1048, 1054, 1055, 1072, 1073], 

  'date': '2004-12-25', 

  'raw': "[03:57] <Xophe> (also, I'm guessing that this isn't a good place to report minor but annoying bugs... what is?)", 

  'tokenized': "<s> ( also , i 'm guessing that this is n't a good place to report minor but annoying bugs ... what is ?) </s>"

}
```
````